Library/Formula/agda.rb

```diff
@@ -20,10 +18,13 @@ class Agda < Formula
   head "https://github.com/agda/agda.git", :branch => "master"
 
   option "without-stdlib", "Don't install the Agda standard library"
-  option "with-malonzo-ffi",
-    "Include the MAlonzo backend's FFI (depends on the standard library)"
+  option "without-ghc", "Don't install GHC (needed for the MAlonzo backend)"
 
-  depends_on "ghc" => :build
+  if build.with? "ghc"
+    depends_on "ghc"
```

Comment:
The failures quoted above do not happen at runtime. In these failures, `cabal sandbox init` fails due to `ghc` not being available at _build_ time.